### PR TITLE
Placeholders for VPC/VPN actions

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Comment by henrymollman** [comment]
The subaction for VPC and VPN are both none, and the Properties can be VPC/VPN since the payload has the same properties as the VPC/VPN object itself.

For example, to create a VPC, the payload in audit log was `{"name":"henry-test-vpc-cidrs","region":"us-west-1","cidrBlock":"10.0.0.0/16","internalDns":{"dnsServers":[],"specialDomains":[]},"isDefault":false,"associatedEnvironments":["915a44c8-6ff6-4746-be24-fa97d0d54654"],"ownerId":"0346a3f1-fc9b-4988-b52c-2f185df42b13","sharedWith":[],"firewallRules":[{"cidrBlock":"0.0.0.0/0","protocol":"tcp","fromPort":8081,"toPort":8081},{"cidrBlock":"0.0.0.0/0","protocol":"tcp","fromPort":8082,"toPort":8082},{"cidrBlock":"10.0.0.0/16","protocol":"tcp","fromPort":8091,"toPort":8091},{"cidrBlock":"10.0.0.0/16","protocol":"tcp","fromPort":8092,"toPort":8092}],"default":false}`

while the update payload was 

`{"id":"vpc-0882c78e66bcb0f1c","name":"henry-test-vpc-cidrs","region":"us-west-1","cidrBlock":"10.0.0.0/16","internalDns":{"dnsServers":[],"specialDomains":[]},"isDefault":false,"associatedEnvironments":["915a44c8-6ff6-4746-be24-fa97d0d54654"],"ownerId":"0346a3f1-fc9b-4988-b52c-2f185df42b13","sharedWith":[],"firewallRules":[{"cidrBlock":"0.0.0.0/0","protocol":"tcp","fromPort":8081,"toPort":8081},{"cidrBlock":"0.0.0.0/0","protocol":"tcp","fromPort":8082,"toPort":8082},{"cidrBlock":"10.0.0.0/16","protocol":"tcp","fromPort":8091,"toPort":8091},{"cidrBlock":"10.0.0.0/16","protocol":"tcp","fromPort":8092,"toPort":8092}],"vpcRoutes":[{"Next Hop":"Local","CIDR":"10.0.0.0/16"},{"Next Hop":"Internet Gateway","CIDR":"0.0.0.0/0"}],"default":false}`

While for VPN the create was

`{"vpcId":"vpc-068b882d34d01abec","vpnId":"601c4e1e0aa6bf36bd6be10c","body":{"name":"test-1234","remoteIpAddress":"99.9.99.9","tunnelConfigs":[],"remoteNetworks":["192.168.0.0/24","98.0.0.1/32"]}`

and modify was

`{"vpcId":"vpc-068b882d34d01abec","vpnId":"601c4e1e0aa6bf36bd6be10c","body":{"name":"test-1234","remoteIpAddress":"99.9.99.9","tunnelConfigs":[],"remoteNetworks":["10.10.0.0/16","192.168.0.0/24","98.0.0.1/32"]}`

For Deletion there are no properties necessary, just the Ids in the URI

**Review by henrymollman — DISMISSED** [review]

**Review by nbedarkar — DISMISSED** [review]

File: modules/ROOT/pages/audit-logging.adoc [hunk 1]
```diff
@@ -1764,7 +1765,20 @@ Properties: LoadBalancer metadata
...
+|Create/modify/delete
+|Subaction: None +
+Properties: VPC
+|Create/modify/delete VPN
+|VPN
+|vpnId
+|N/A
+|Create/modify/delete
+|Subaction: None +
+Properties: VPN
```
Comment (nbedarkar) [inline modules/ROOT/pages/audit-logging.adoc]: change to "Properties: VPC Metadata"?

File: modules/ROOT/pages/audit-logging.adoc [hunk 2]
```diff
@@ -1764,7 +1765,20 @@ Properties: LoadBalancer metadata
...
 |Subaction: None +
 Properties: Alert
-
+|Create/modify/delete VPC
+|VPC
+|vpcID
+|N/A
+|Create/modify/delete
+|Subaction: None +
+Properties: VPC
```
Comment (nbedarkar) [inline modules/ROOT/pages/audit-logging.adoc]: change to "Properties: VPC Metadata"?

**Review by henrymollman — APPROVED** [review]

**Review by mcherem — APPROVED** [review]
Peer review.

**Pull request merged**